installer/osie-installer.sh

```diff
@@ -173,6 +174,7 @@ docker run --privileged -ti \
 	-h "${hardware_id}" \
 	-e "container_uuid=$id" \
 	-e "RLOGHOST=$tinkerbell" \
+	-e "ECLYPSIUM_TOKEN=$eclypsium_token" \
```

Comment:
this needs to be `${eclypsium_token:-}` otherwise will cause an error when using an unset variable right?

Reply:
The sed parsing of /proc/cmdline will fail to define the env var if eclypsium_token is not present, regardless if it has an empty value or not.    Subsequently when docker run is called, the '-e ECLYPSIUM_TOKEN=' parameter will not cause an error, and will pass the 'empty' variable into the container.    That being said, I agree with being more explicit, so I have made the suggested change.